Riot/Modules/Home/VersionCheck/VersionCheckCoordinator.swift

```diff
@@ -0,0 +1,128 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+private let osVersionToBeDropped = 11
+private let hasOSVersionBeenDropped = false
```

Comment:
Please use constants inside the `VersionCheckCoordinator` class as they are used outside of this context:
```
private enum Constants {
  static let osVersionToBeDropped = 11
}
```
Using a `VersionCheckService` to handle only the data logic is also another option.